VaticanBAV/et/BAVet155.xml

```diff
@@ -26,140 +26,200 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        </item>
+                                        <item xml:id="q4" n="3">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="23r" to="32v"/>
+                                        </item>
+                                        <item xml:id="q5" n="4">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="33r" to="42v"/>
+                                        </item>
+                                        <item xml:id="q6" n="5">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="43r" to="50v"/>
+                                        </item>
+                                        <item xml:id="q7" n="6">
+                                            <dim unit="leaf">6</dim>
+                                            <locus from="51r" to="56v"/>
+                                        </item>
+                                    </list>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout writtenLines="18"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <date notBefore="1700" notAfter="1799">18th century</date>
+                                <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">
+                                    Titles have not been added in red ink from <locus target="#29v"/> onwards, resulting in unfilled blank spaces.
+                                </seg>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#53v"/>
+                                    <desc type="ScribalSignature">Note mentioning the name of the scribe, <persName ref="PRS15081WaldaSa">Walda Ṣādǝq</persName>.</desc>
+                                    <q xml:lang="gez">ብርዕ፡ ሠናይ፡ ዘፈተነ፡ ቀለመ፡ <choice><corr resp="bm:GrebTiss1935Codices">ጸሐፊሁ፡</corr><sic>ጸሐፈሁ፡</sic></choice>
+                                        <persName ref="PRS15081WaldaSa">ወ<add place="above">ል</add>ደ፡ ጻድቅ፡</persName></q>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1">Two wooden boards covered.</decoNote>
```

Comment:
This "covered" sounds to me strange, as if something was missing (covered with what?)